src/composables/contextMenu/useCopyLink.ts

```diff
@@ -9,12 +9,12 @@ const useCopyLink = (messageId: Ref<MessageId>) => {
 
   const copyLink = async () => {
     const link = `${embeddingOrigin}${constructMessagesPath(messageId.value)}`
-    await copyText(link)
+    await copyText(link, 'メッセージリンク')
   }
 
   const copyEmbedded = async () => {
     const link = `<iframe src="${embeddingOrigin}/widget/?type=message&id=${messageId.value}" scrolling="no" frameborder="no" width="600"></iframe>`
-    await copyText(link)
+    await copyText(link, '埋め込み')
```

Comment:
「埋め込み」をコピーだとちょっと日本語に違和感を覚えてしまったので、こんな感じにしてみてもいいと思います
ただ、ちょっと長いのでお任せします
```suggestion
    await copyText(link, 'メッセージ埋め込みテキスト')
```

Reply:
あんまり長くしてもスマホで場所を取る&長くしたところで何がコピーされたかは伝わらなさそう(いい文言が思いつかない)なのでこのままでいいのかなと思います。
日本語的におかしいっていうのはたしかにそうで、めちゃくちゃいい指摘だと思います。:pro:
ただ今回はコピーされるものを端的に表す単語がないという状況なので、正確性よりかは短さを優先しちゃおうかと思います。

Reply:
:haakusimasita: :yosasou: :desu: